clang/test/Driver/android-link.cpp

```diff
@@ -17,10 +17,20 @@
 //
 // RUN: %clang -target aarch64-none-linux-android \
 // RUN:   -### -v %s 2> %t
-// RUN: FileCheck -check-prefix=MAX-PAGE-SIZE-AARCH64 < %t %s
+// RUN: FileCheck -check-prefix=MAX-PAGE-SIZE-16KB < %t %s
+//
+// RUN: %clang -### -target i386-none-linux-android %s 2>&1 | \
+// RUN:   FileCheck --check-prefix=NO-MAX-PAGE-SIZE-16KB %s
+//
```

Comment:
Don't add `^//$` lines

Reply:
Removed `^//$` lines. Thanks for reviews.